WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.kt

```diff
@@ -267,66 +242,84 @@ class MySiteFragment : Fragment(R.layout.my_site_fragment),
+        is SiteNavigationAction.OpenComments -> ActivityLauncher.viewCurrentBlogComments(activity, action.site)
+        is SiteNavigationAction.OpenUnifiedComments -> ActivityLauncher.viewUnifiedComments(activity, action.site)
+        is SiteNavigationAction.OpenStats -> ActivityLauncher.viewBlogStats(activity, action.site)
+        is SiteNavigationAction.ConnectJetpackForStats ->
+            ActivityLauncher.viewConnectJetpackForStats(activity, action.site)
+        is SiteNavigationAction.StartWPComLoginForJetpackStats ->
+            ActivityLauncher.loginForJetpackStats(this@MySiteFragment)
+        is SiteNavigationAction.OpenJetpackSettings ->
+            ActivityLauncher.viewJetpackSecuritySettings(activity, action.site)
+        is SiteNavigationAction.OpenStories -> ActivityLauncher.viewStories(activity, action.site, action.event)
+        is SiteNavigationAction.AddNewStory ->
+            ActivityLauncher.addNewStoryForResult(activity, action.site, action.source)
+        is SiteNavigationAction.AddNewStoryWithMediaIds -> ActivityLauncher.addNewStoryWithMediaIdsForResult(
                 activity,
                 action.site,
                 action.source,
                 action.mediaIds.toLongArray()
         )
-        is AddNewStoryWithMediaUris -> ActivityLauncher.addNewStoryWithMediaUrisForResult(
+        is SiteNavigationAction.AddNewStoryWithMediaUris -> ActivityLauncher.addNewStoryWithMediaUrisForResult(
                 activity,
                 action.site,
                 action.source,
                 action.mediaUris.toTypedArray()
         )
-        is OpenDomains -> ActivityLauncher.viewDomainsDashboardActivity(
+        is SiteNavigationAction.OpenDomains -> ActivityLauncher.viewDomainsDashboardActivity(
                 activity,
                 action.site
         )
-        is OpenDomainRegistration -> ActivityLauncher.viewDomainRegistrationActivityForResult(
+        is SiteNavigationAction.OpenDomainRegistration -> ActivityLauncher.viewDomainRegistrationActivityForResult(
                 activity,
                 action.site,
                 CTA_DOMAIN_CREDIT_REDEMPTION
         )
-        is AddNewSite -> SitePickerActivity.addSite(activity, action.isSignedInWpCom)
-        is ShowQuickStartDialog -> showQuickStartDialog(
+        is SiteNavigationAction.AddNewSite -> SitePickerActivity.addSite(activity, action.isSignedInWpCom)
+        is SiteNavigationAction.ShowQuickStartDialog -> showQuickStartDialog(
                 action.title,
                 action.message,
                 action.positiveButtonLabel,
                 action.negativeButtonLabel
         )
-        is OpenQuickStartFullScreenDialog -> openQuickStartFullScreenDialog(action)
+        is SiteNavigationAction.OpenQuickStartFullScreenDialog -> openQuickStartFullScreenDialog(action)
+        is SiteNavigationAction.OpenDrafts ->
```

Comment:
1. Minor (🔍): Since the below data class is named as `OpenScheduledPosts` consider renaming this to `OpenDraftsPosts`, or vice-versa, renaming the `OpenScheduledPosts` to ``OpenScheduled`, without the `Posts` keyword at the end, whatever you prefer, but keeping consistency on both.
2. Minor (🔍): I noticed that the `val postListType: PostListType` is unused on both the `OpenDraftsPosts` and `OpenScheduledPosts`, probably because initially you might wanted to have both data classes as one, but then decided to split it. Consider removing this field or merge these two data classes into one, whichever you prefer.

Reply:
1. I matched the name with the footer link `Open drafts` and `Open scheduled posts`. I renamed them as per suggestion, they need not match the language on the footer link and we can prefer consistency. 🙂
2. Nice find 👍, yes I was trying to do something similar. 😄 

In dc9eecd

Reply:
👋 @ashiagr !

> I matched the name with the footer link Open drafts and Open scheduled posts. 

Oh, I now see why this was named the way it was, thanks for explaining, it is so very interesting to understand different ways of thinking! 👍